### PR TITLE
Minor typos

### PR DESCRIPTION
**Pull request by skyler1ackerman**
Just a quick couple of issues I noticed when running through the docs.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -406,7 +406,7 @@ Vantiq Document. Providing both `filename` and `doc_name` in this case is an err
 
 ### Examples
 ```python
-vr: VantiqResponse = await client.upload(VantiqResources.DOCUMENTS, 'iamge/png', '/file/name.png')
+vr: VantiqResponse = await client.upload(VantiqResources.DOCUMENTS, 'image/png', 'file/name.png')
```
Comment (fhcarter) [inline docs/api.md:409]:
```suggestion
vr: VantiqResponse = await client.upload(VantiqResources.DOCUMENTS, 'image/png', 'path/name.png')
```

**Review by steve-langley — APPROVED** [review]
LGTM

File: docs/api.md [hunk 2]
```diff
@@ -406,7 +406,7 @@ Vantiq Document. Providing both `filename` and `doc_name` in this case is an err
...
 if vr.is_success:
     # Here, we will have uploaded a file named `/file/name.png`,
```
Comment (fhcarter) [inline docs/api.md:411]: should change here, too (or I can just add all of these to the count PR with the insert/update/upsert stuff, etc.

**Review by fhcarter — APPROVED** [review]
LGTM

**Pull request merged**